packages/codemods/react-router/4/replace-nested-routes/src/index.ts

```diff
@@ -0,0 +1,80 @@
+import type { API, FileInfo, JSXElement } from "jscodeshift";
+
+function transform(file: FileInfo, api: API): string | undefined {
+	const j = api.jscodeshift;
+	const root = j(file.source);
+
+	const routes = root.find(j.JSXElement, {
+		openingElement: {
+			name: {
+				name: "Route",
+			},
+		},
+		closingElement: {
```

Comment:
what if Route is selfclosing jsx tag?

Reply:
We are looking for Route components that nest Route components